pkgs/development/tools/parsing/tree-sitter/grammar.nix

```diff
@@ -30,7 +30,11 @@ stdenv.mkDerivation {
     if [ ! -f "$scanner_cc" ]; then
       scanner_cc=""
     fi
-    $CC -I$src/src/ -shared -o parser -Os $src/src/parser.c $scanner_cc -lstdc++
+    scanner_c="$src/src/scanner.c"
+    if [ ! -f "$scanner_c" ]; then
+      scanner_c=""
+    fi
+    $CC -I$src/src/ -shared -o parser -Os $src/src/parser.c $scanner_cc $scanner_c -lstdc++
```

Comment:
One question I have is: will there ever be a case of a scanner providing both scanner.c and scanner.cc? If so, do we want both of them, or either/or?